.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -30,7 +30,7 @@ Call-to-action statement describing what you are doing in this PR.
 
 ### Other information
 <!--
-    Are there other relevant information, such as special testing instructions, 
+    Are there other relevant information, such as special testing instructions,
```

Comment:
**To reviewers:** 
Changes made to files under `.github` and `docs` are only formatting changes and can be ignored.